From e2faee2852290f8eb482760f21ceed27f147e2da Mon Sep 17 00:00:00 2001
From: Tong Bo <bo.tong@intel.com>
Date: Thu, 24 May 2018 17:37:24 +0800
Subject: [PATCH 2/5] [REVERTME] Add vendor/{LIB}/hw as default vendor lib
 search path

Some binderized HALs(e.g. Evs and BroadcastRadio) require linking
shared libraries(i.e., "-impl" libs) under vendor/{LIB}/hw.

There are other ways to resolve this linking issue.

1)Move relevant shared libraries under vendor/{LIB}/hw to
vendor/{LIB}, which is already a default vendor lib search path.

2)Convert the binderized HALs to passthrough mode, and there is
an API called defaultPassthroughServiceImplementation() that will
dlopen() shared libraries with an absolute path.

Change-Id: Ia2a0e134038a0ac48fe2c35890a60df2306740b7
Tracked-On:
Signed-off-by: Tong Bo <bo.tong@intel.com>
---
 rootdir/etc/ld.config.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rootdir/etc/ld.config.txt b/rootdir/etc/ld.config.txt
index 9722851ac..b40d21b1d 100644
--- a/rootdir/etc/ld.config.txt
+++ b/rootdir/etc/ld.config.txt
@@ -402,6 +402,7 @@ namespace.default.visible = true

 namespace.default.search.paths  = /odm/${LIB}
 namespace.default.search.paths += /vendor/${LIB}
+namespace.default.search.paths += /vendor/${LIB}/hw

 namespace.default.permitted.paths  = /odm
 namespace.default.permitted.paths += /vendor
--
2.21.0
